information-schema: add tidb_hot_regions_history docs and update tidb_hot_regions (#6624)
pingcap/docs: information-schema/information-schema-tidb-hot-regions.md

# TIDB_HOT_REGIONS

> The `TIDB_HOT_REGIONS` table provides information about current hotspot Regions, history hotspot is available in `[TIDB_HOT_REGIONS_HISTORY](/information-schema/information-schema-tidb-hot-regions-history.md)`.

Reviewer: ```suggestion
The `TIDB_HOT_REGIONS` table provides information about the current hot Regions. For information about history hot regions, see `[TIDB_HOT_REGIONS_HISTORY](/information-schema/information-schema-tidb-hot-regions-history.md)`.
```

> The `TIDB_HOT_REGIONS` table provides information about the current hot Regions. For information about history hot regions, see `[TIDB_HOT_REGIONS_HISTORY](/information-schema/information-schema-tidb-hot-regions-history.md)`.

Reviewer 2: ```suggestion
The `TIDB_HOT_REGIONS` table provides information about the current hot Regions. For information about history hot Regions, see `[TIDB_HOT_REGIONS_HISTORY](/information-schema/information-schema-tidb-hot-regions-history.md)`.
```
